website/docs/cloud-docs/migrate/index.mdx

```diff
@@ -1,43 +1,79 @@
 
-> **Hands-on:** Try the [Migrate State to HCP Terraform](/terraform/tutorials/state/cloud-migrate) tutorial to practice using the CLI method.
+## Overview
+
+Perform the following actions to migrate existing resources to one or more workspaces in HCP Terraform or Terraform Enterprise:
+
+1. Stop all Terraform operations associated with the state files. 
+1. Migrate Terraform state files using either the Terraform CLI or the HCP Terraform or Terraform Enterprise API.
+
+> **Hands-on:** Complete the [Migrate State to HCP Terraform](/terraform/tutorials/state/cloud-migrate) tutorial for additional guidance on how to migrate Terraform state using the CLI.
 
 ## Requirements
 
-Stop all Terraform operations involving the state files before migrating them. This may involve locking or deleting CI jobs, restricting access to the state backend, or communicating with other teams. You should also only migrate state files into HCP Terraform workspaces that have never performed a run.
+- Terraform v1.1 and later is required to configure the `cloud` block. For Terraform v1.0 and older, use the [`remote` backend](/terraform/language/settings/backends/remote) instead.
+- You must present a token linked to appropriate permissions to use the API. Refer to the following topics for additional information:
+  - [HCP Terraform API overview](/terraform/cloud-docs/api-docs)
+  - [Terraform Enterprise API overview](/terraform/enterprise/api-docs)
 
-## CLI Migration
+## Stop Terraform operations
 
-To migrate with the Terraform CLI, add the `cloud` block to your configuration, specify one or more HCP Terraform workspaces for the state files, and run `terraform init`. If the workspaces you choose do not yet exist, HCP Terraform creates them automatically in the specified organization.
+Stop all Terraform operations associated with the state files. This may require locking or deleting CI jobs, restricting access to the state backend, and communicating with other teams. You should also only migrate state files into HCP Terraform or Terraform Enterprise workspaces that have never performed a run.
 
-The example below shows how you can map your CLI workspaces to HCP Terraform workspaces that have a specific tag.
+## Migrate state using the CLI
 
-```
-terraform {
-  cloud {
-    organization = "my-org"
-    workspaces {
-      tags = ["networking"]
+1. Add the `cloud` block to your Terraform configuration and specify the following fields:
+   1. `hostname` field: Specify either `app.terraform.io` for HCP Terraform or the hostname of your Terraform Enterprise deployment. 
+   1. `organization` field: Specify your HCP Terraform or Terraform Enterprise organization. 
+   1. `workspaces` block: Add the `tags` field and specify one or more destination workspaces as a list of strings. 
```

Comment:
Should we mention that the `workspaces` block can defined the `tags` field OR the `name` field? I did some digging around and it looks like `tags` is the recommended way, according to this CLI output:

```
│ Only one of workspace "tags" or "name" is allowed.
│
│ The 'workspaces' block configures how Terraform CLI maps its workspaces for this single
│ configuration to workspaces within an HCP Terraform or Terraform Enterprise organization. Two
│ strategies are available:
│
│ tags - A set of tags used to select remote HCP Terraform or Terraform Enterprise
│ workspaces to be used for this single
│ configuration. New workspaces will automatically be tagged with these tag values. Generally, this
│ is the primary and recommended strategy to use.  This option conflicts with "name".
│
│ name - The name of a single HCP Terraform or Terraform Enterprise workspace to be
│ used with this configuration.
│ When configured, only the specified workspace can be used. This option conflicts with "tags"
│ and with the TF_WORKSPACE environment variable.
```

Might just be how I read it, but this makes me feel like I can _only_ use the `tags` field when migrating my state.